WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEpilogueFragment.java

```diff
@@ -116,6 +116,10 @@ private int loginEpilogueScreenResource() {
         }
     }
 
+    private boolean newLoginEpilogue() {
```

Comment:
@ParaskP7 
❓  `newLoginEpilogue` - can we come up with a more descriptive name. Our future selves will thank us.

Reply:
👋 @zwarm !

Thank you so much for this comment, I was actually looking forward to it! 🌟 

I spent 5' mins (ish) in thinking what would the best name be for this, but after a while, and me trying a few thing, I ended up decided on giving it a name not by the `What` it does principle, but by the `Why` it does what it does principle. My problem with that method was that it is used in a number of places and although giving a name like `shouldUseNewLoginEpilogueScreen()` might be a good fit for this one usage, it would not be a good fit for all the other usages, and vice-versa. As such, I decided to keep the naming simple using a common denominator.

I know we have discussed about changing the `New` to something else, since this is not actually a new `Epilogue Screen`, but another one and both the `Existing` and this `Another` one will be anyway both used, even when we remove the `Onboarding Improvements` feature flag, But, since everything is still named with either a `New` suffix or prefix, I chose to go with that. On a separate refactor PR we can change all that to a more appropriate naming, after we sync between us and decided what would the best naming be.

Also, I chose to stick to the `newLoginEpilogue()` naming for now since my next `Alternative Flow` PR will add to this signature and I wasn't sure how this might require a name change yet again. Thus, I want to defer this naming decision for last.

👋 @ashiagr , I am shamelessly pinging you on this as well, because:
1. Maybe you can suggest a good naming to us, and
2. So that you are aware of our already generic naming related discussion with @zwarm so that we can sync on it later on and decide on something more appropriate overall.

PS: When I was trying on different names I also tried the following:
- `isNewLoginEpilogue()`
- `withNewLoginEpilogue()`
- `whenForNewLoginEpilogue()`
- `shouldUseNewLoginEpilogueScreen()`
- ...and some others I didn't quite end up liking.
Please suggest me a better alternative and I'll be happy to rename it! 🙏

Reply:
Thanks for the ping @ParaskP7! Naming is really hard 😅. 

Based on the conditions included inside the method, we might want to rename it to  `shouldShowNewLoginEpilogueScreen()` but let's defer this decision till we've finalized all changes and are ready to remove the `new` prefix or suffix used for the feature.

Reply:
Naming is hard indeed! 😅 

Thanks for the suggestion @ashiagr , this `shouldShowNewLoginEpilogueScreen()` sounds great, but it still falls under this problem that I was having where although this is the correct name when used for this main usage:
```
if (shouldShowNewLoginEpilogueScreen()) {
    return R.layout.login_epilogue_screen_new;
} else {
    return R.layout.login_epilogue_screen;
}
```
This doesn't feel correct when used on some other usages, like the:
```
if (shouldShowNewLoginEpilogueScreen()) {
    mLoginEpilogueListener.onCreateNewSite();
} else {
    mLoginEpilogueListener.onContinue();
}
```
This is because on the one case is about deciding which screen to show, where the name totally makes sense, but on the other case is about deciding what action to do based on the screen, not asking if we want to `Show` the screen. Do you see my problem with this naming?

PS: No matter what, if you both feel that we can and should use the `shouldShowNewLoginEpilogueScreen()`, I am 💯 onboard. @zwarm your thoughts?

Reply:
I see. How about `isNewLoginEpilogueScreenEnabled()`?

Reply:
I like that much better yes! 🌟

Reply:
@zwarm I am going to go with what @ashiagr suggested for now and update the PR, also merge the latest `develop` changes and resolve any conflicts. Then, if we decided on a better name we can incorporate that on my next PR (expected today).

Reply:
This is now done here: 429b230634baf00587152d550d6bab498a62479d

Reply:
👍  I'm onboard ... naming is **_hard_** . LOL.